data-plane/receiver-loom/src/main/java/dev/knative/eventing/kafka/broker/receiverloom/LoomKafkaProducer.java

```diff
@@ -57,7 +57,8 @@ public LoomKafkaProducer(Vertx v, Producer<K, V> producer) {
         final var ctxInt = ((ContextInternal) v.getOrCreateContext()).unwrap();
         if (ctxInt.tracer() != null) {
             this.tracer =
-                    new ProducerTracer(ctxInt.tracer(), TracingPolicy.PROPAGATE, "" /* TODO add bootrstrap servers */);
+                    new ProducerTracer(this.vertx.tracer(), TracingPolicy.PROPAGATE, "" /* TODO add bootrstrap servers */);
```

Comment:
@tsegismont Even changing this, still is causing the `ReceiverVerticleTracingTest` unit test to fail